assets/sass/elements/_forms.scss

```diff
@@ -167,12 +158,45 @@ textarea.form-control {
 }
 
 textarea.form-control {
+  background-image: none;
+  // Remove margin under textarea in Chrome and FF
+  display: block;
   // Disable opacity and background image for Firefox
   opacity: 1;
-  background-image: none;
 }
-// scss-lint:enable QualifyingElement
 
+select.form-control {
+  -moz-appearance: none;
+  -webkit-appearance: none;
+  appearance: none;
+  background-color: $white;
+  background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAYCAYAAACbU/80AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAIpJREFUeNrEkckNgDAMBBfRkEt0ObRBBdsGXUDgmQfK4XhH2m8czQAAy27R3tsw4Qfe2x8uOO6oYLb6GlOor3GF+swURAOmUJ+RwtEJs9WvTGEYxBXqI1MQAZhCfUQKRzDMVj+TwrAIV6jvSUEkYAr1LSkcyTBb/V+KYfX7xAeusq3sLDtGH3kEGACPWIflNZfhRQAAAABJRU5ErkJggg==');
+  background-position: center right 10px;
```

Comment:
Is that a typo? 3 values means translated to 4 values it's "center right 10px right". That doesn't look right.